cypress/support/commands.ts

```diff
@@ -94,15 +93,15 @@ Cypress.Commands.add(
 
 Cypress.Commands.add("useAutocomplete", (newShiftCode: ShiftCode) => {
   return cy
-    .get(`[data-cy=autocomplete-${newShiftCode}]`, { timeout: 100000 })
+    .get(`[data-cy=autocomplete-${newShiftCode}]`, { timeout: 10000 })
     .click({ force: true });
 });
 
 Cypress.Commands.add(
   "changeWorkerShift",
   ({ newShiftCode, ...getWorkerShiftOptions }: ChangeWorkerShiftOptions) => {
     cy.getWorkerShift(getWorkerShiftOptions).click();
-    return cy.useAutocomplete(newShiftCode);
```

Comment:
Dlaczego usunąłeś return?
Mi się wydaje, że nie przeszkadzał, natomiast pozwalał nam na to, żeby używać to polecenie w łańcuchu. 

Taki przykładowy przypadek użycia który teraz przychodzi mi do głowy to test w którym najpierw chcielibyśmy zmienić zmianę raz, sprawdzić, a potem to przywrócić.

Reply:
Dlatego, że w useAutocomplete jest zwracany click() i wydawało mi niepotrzebne zwracanie tego, ale faktyczni lepiej jest to zwracać, bo przecież możemy chcieć coś dodatkowo zrobić i tracimy to co nam daje Chainable z cypressa. 

Przywrócę tego returna